web/src/utils/formatting.test.ts

```diff
@@ -23,27 +23,42 @@ describe('formatCo2', () => {
     const expected = '0 t';
     expect(actual).toBe(expected);
   });
+  it('uses same unit as another value would non-zero', () => {
+    const actual = formatCo2(230_500, 2_350_000);
+    const expected = '0.2 t';
+    expect(actual).toBe(expected);
+  });
 
   it('adds decimals if comparing with tons', () => {
-    const actual = formatCo2(12_003_500, 200_350_000);
+    const actual = formatCo2(200_500, 2_350_000);
     const expected = '0.2 t';
     expect(actual).toBe(expected);
   });
   it('handles values up to 100k', () => {
-    const actual = formatCo2(700_003_500_000);
-    const expected = '11,667 t';
+    const actual = formatCo2(99_000_000_000_000_000);
+    const expected = '99 Gt';
     expect(actual).toBe(expected);
   });
-  it('handles values up to 1 million', () => {
-    const actual = formatCo2(30_000_035_000_000);
-    const expected = '0.5 Mt';
+  it('handles values up to megatonnes', () => {
+    const actual = formatCo2(500_000_000_000_000);
+    const expected = '500 Mt';
     expect(actual).toBe(expected);
   });
-  it('handles values above 1 million', () => {
-    const actual = formatCo2(400_000_350_000_000_000);
+  it('handles values up to 1 trillion grams', () => {
+    const actual = formatCo2(999_000_000_000_000);
+    const expected = '1 Gt';
+    expect(actual).toBe(expected);
+  });
```

Comment:
Maybe put this number lower so that it is not rounded up to Gt - ideally it should return 0.X Gt

Reply:
I think I've updated the tests now to cover all obvious edge cases, it's a little hard to know what 100k means now we've changed to grams as the base unit. Could you take a look at the formatting tests in their current state and give further feedback?

Reply:
Fair point, will look through it all after we merge this PR :)